packages/agent/esbuild.mjs

```diff
@@ -12,7 +12,7 @@ const options = {
   resolveExtensions: ['.js', '.ts'],
   target: 'es2021',
   tsconfig: 'tsconfig.json',
-  external: ['iconv-lite', 'pdfmake'],
```

Comment:
In the Node single executable we can't keep `iconv-lite` because it doesn't do any of the auto-bundling magic that `pkg` did and only works with a single bundled file. `iconv-lite` is a completely JS module so this works though we may have to figure out how native modules work in the Node SEA env since I don't think it's quite as "magical" as the `pkg` version, though we saw that `pkg` itself had its issue with native stuff